### DISCUSSION TITLE
Kalliope - Github snowboy dead

### DISCUSSION THREAD

**Discussion by Xorck** [community/community#56021] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Bonjour, 

J'ai une question, je n'arrive pas à lancer mon kalliope en raison de l'absence de snowboy. Impossible de le telecharger car il n'est plus actif. Avez-vous une solution ?

**Comment by queenofcorgis** [comment]
Hi @Xorck ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Kalliope or Snowboy. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
